front/app/containers/Admin/projects/project/events/edit.tsx

```diff
@@ -390,10 +393,24 @@ class AdminProjectEventEdit extends PureComponent<Props, State> {
   }
 }
 
-export default (props: InputProps) => (
-  <GetRemoteFiles resourceId={props.params.id} resourceType="event">
-    {(remoteEventFiles) => (
-      <AdminProjectEventEdit remoteEventFiles={remoteEventFiles} {...props} />
+const Data = adopt<Props>({
```

Comment:
I could not figure out how the `project` was being passed in as a prop in the old component, so I updated it to this. It works fully in my local testing but if there's an easier way let me know